test/test_quickstart.jl

```diff
@@ -5,14 +5,14 @@ using RDatasets: dataset
 
 iris = dataset("datasets", "iris")
 
-X = convert(Array, iris[[:SepalLength, :SepalWidth, :PetalLength, :PetalWidth]])
-y = convert(Array, iris[:Species])
+X = convert(Array, iris[!, [:SepalLength, :SepalWidth, :PetalLength, :PetalWidth]])
```

Comment:
What are these for?

Reply:
It fixes a depreciation warning from the dataframes package